### PR TITLE
Friend List feature

### PR DESCRIPTION
**Pull request by cjinting-nus**
Creating PR first to prevent bigbang

- [x] Created a friendlist
- [x] FriendList able to display on UI
- [x] Modify logic for friendList to display friends
- [x] FriendCommand
- [x] UnfriendCommand

### CODE REVIEW

**Comment by bennchong** [comment]
Sick la

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
     @Override
     public void init() throws Exception {
-        logger.info("=============================[ Initializing AddressBook ]===========================");
+        logger.info("=============================[ Initializing FreeTime ]===========================");
```
Comment (nianfei97) [inline src/main/java/seedu/address/MainApp.java]: Thanks for updating the name of the app

File: src/main/java/seedu/address/model/person/OtherListPredicate.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import java.util.Set;
+import java.util.function.Predicate;
+
+/**
+ * Creates a predicate to be used to find the non-friends in persons list
+ */
+public class OtherListPredicate implements Predicate<Person> {
+
+    private final Set<Friend> otherList;
```
Comment (nianfei97) [inline src/main/java/seedu/address/model/person/OtherListPredicate.java]: This kinda doesn't make sense... So are these people friends, or not friends?

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 3]
```diff
@@ -126,12 +141,27 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      * Fills up all the placeholders of this window.
      */
     void fillInnerParts() {
+
+        friendText.setText("Friends");
```
Comment (nianfei97) [inline src/main/java/seedu/address/ui/MainWindow.java]: Something to be done next milestone is probably to implement this with a .css stylesheet instead

File: src/main/java/seedu/address/ui/UiManager.java [hunk 4]
```diff
@@ -59,7 +59,7 @@ public void start(Stage primaryStage) {
             mainWindow = new MainWindow(primaryStage, config, prefs, logic, security);
             mainWindow.show(); //This should be called before creating other UI parts
             //Gets rid of the login window when testing
-            if (security.getAuthentication()) {
+            if (true) { // TODO : security.getAuthentication()
                 mainWindow.fillInnerParts();
```
Comment (nianfei97) [inline src/main/java/seedu/address/ui/UiManager.java]: There has to be a better way to handle skipping the login screen... Hopefully we remember to remove this before v1.3

**Pull request merged**